pipeline-scripts/create-database.sql

```diff
@@ -628,6 +629,171 @@ CREATE PROCEDURE CreateDatabase ()
                 ON DELETE CASCADE
         ) ENGINE=INNODB;
 
+        CREATE TABLE IF NOT EXISTS vault_governance
+        (
+            Id                  BIGINT UNSIGNED AUTO_INCREMENT,
+            TokenId             BIGINT UNSIGNED NOT NULL,
+            Address             VARCHAR(50)     NOT NULL,
+            UnassignedSupply    VARCHAR(78)     NOT NULL,
+            VestingDuration     BIGINT UNSIGNED NOT NULL,
+            ProposedSupply      VARCHAR(78)     NOT NULL,
+            PledgeMinimum       BIGINT UNSIGNED NOT NULL,
+            ProposalMinimum     BIGINT UNSIGNED NOT NULL,
```

Comment:
Might want to rename - see https://github.com/Opdex/opdex-governance/pull/6#discussion_r760903901_